usage-examples/submodules-eiffel.md

```diff
@@ -0,0 +1,18 @@
+# Working with GIT submodules
+This example discusses how one could build the relational tree structure representing submodules within GIT using Eiffel events.
+
+## Introduction
+A common use case in GIT is where one want to connect a relationship between repositories to be able to mirror the GIT structure as it is. The scenario could be that you have a parent repository who have one or many GIT submodule.
+
+Below is a simple image of how it could look in GIT.
+
+![alt text](./submodule-git.png)
+
+## Eiffel solution
+This will show you one setup for the Eiffel model on how to get around the use-case where there is a sceanrio where GIT submodules are used.
```

Comment:
```suggestion
This will show you one setup for the Eiffel model on how to get around the use-case where there is a scenario where GIT submodules are used.
```